packages/radio-button-react/example/index.tsx

```diff
@@ -13,17 +15,37 @@ const Demo = () => {
     const [selectedValue, setSelectedValue] = React.useState();
     const [inline, setInline] = useState(false);
     const [invalid, setInvalid] = useState("");
+    const [variant, setVariant] = useState<labelVariant | undefined>();
+    const typecheckAndSetVariant = (e: React.ChangeEvent<HTMLSelectElement>) => {
+        const val = e.target.value;
+        if (val === ("secondary" || "small")) {
```

Comment:
This will never be true for `small`, will it?

Reply:
I chould just use the type here (it is a typecheck after all) 😅